tests/AttributesAreProvidedInCorrectFormatTest.php

```diff
@@ -71,6 +71,15 @@ public function provideAttributeCombinations(): \Generator
             ],
         ];
 
+        $date = new \DateTime();
+
+        yield [
+            ['Key1' => $date],
+            [
+                ['Name' => 'Key1', 'Value' => $date->format('Y-m-d H:i:s e')],
```

Comment:
Good to have the coverage, great stuff @JoshuaLicense 
Always slightly nervous of new \DateTime in a test but if just checking format all good :-)